packages/get-css/package.json

```diff
@@ -5,32 +5,32 @@
   "main": "index.js",
   "scripts": {
     "format": "prettier --no-semi --single-quote --write {test,utils,bin}/**/*.js index.js",
-    "test": "node test/test && mocha test 'test/**/*.js'"
+    "test": "ava 'test/**/*.js'"
```

Comment:
I was unable to run without using ava, because the tests depend on it, so I also added it to the dev dependencies and remove mocha.